### DISCUSSION TITLE
[Feature request] Iterate items in file

### DISCUSSION THREAD

**Discussion by knighttv** [postmanlabs/postman-flows#434] [Ideas]
As is available in Postman runner, I think it would be a very useful to allow a data source to be an iterable list that can be loaded from disk, such as a JSON or CSV file. This could then pass through the "for" step to produce a trigger per item.

**Comment by viko42** [comment]
Do you mean a flow like this one ?
<img width="854" alt="Screenshot 2023-01-12 at 20 19 45" src="https://user-images.githubusercontent.com/18552774/212172146-92a7e4dd-5aae-4d44-9446-339def6073ad.png">

If you deploy your flow, then you can /POST with any data you want including arrays. and you can iterate over them
